nautobot_golden_config/tests/test_models.py

```diff
@@ -112,12 +113,6 @@ def test_absolute_url_success(self):
         url_string = self.global_settings.get_absolute_url()
         self.assertEqual(url_string, f"/plugins/golden-config/setting/{self.global_settings.slug}/")
 
-    def test_bad_graphql_query(self):
```

Comment:
Was this moved to somewhere else?

Reply:
We no longer need to validate if the string is a valid query as that is now handled by the GraphQLQuery model.